### PR TITLE
Fixed startCopyFromURL and copyFromURL API not respect `--disableProductStyleUrl` in parse source Uri

### PR DESCRIPTION
**Pull request by blueww**
Thanks for contribution! Please go through following checklist before sending PR.
Fix issue https://github.com/Azure/Azurite/issues/1245

### PR Branch Destination

- For Azurite V3, please send PR to `main` branch.
- For legacy Azurite V2, please send PR to `legacy-dev` branch.

### Always Add Test Cases

Make sure test cases are added to cover the code change.

### Add Change Log

Add change log for the code change in `Upcoming Release` section in `ChangeLog.md`.

### Development Guideline

Please go to CONTRIBUTION.md for steps about setting up development environment and recommended Visual Studio Code extensions.

### CODE REVIEW

**Comment by wiruzman** [comment]
Could you link the issue here?

**Comment by blueww** [comment]
@wiruzman 
The issue is already linked.
See the [first comments in the PR](https://github.com/Azure/Azurite/pull/1252#issue-1089029457), it has : Fix issue #1245

**Comment by wiruzman** [comment]
It just do not show up here:
image

**Comment by wiruzman** [comment]
Another question, do you know when this PR is going to be merged?

**Comment by blueww** [comment]
@XiaoningLiu 
Would you please help to look at this PR?

File: src/blob/generated/Context.ts [hunk 1]
```diff
@@ -140,4 +140,12 @@ export default class Context {
   public get startTime(): Date | undefined {
     return this.context.startTime;
   }
+
+  public get disableProductStyleUrl(): boolean | undefined {
```
Comment (XiaoningLiu) [inline src/blob/generated/Context.ts:144]: Please add these 2 methods into `BlobContext` or "Queue|TableContext". All files under "generated" folder shouldn't be manually edited.

**Review by XiaoningLiu — CHANGES_REQUESTED** [review]
One comment added, others look good. Thanks for the update!

File: src/blob/generated/Context.ts (hunk 1 above)
Comment (blueww) [reply src/blob/generated/Context.ts]: fixed

**Comment by blueww** [comment]
@XiaoningLiu 

I have fixed the comment from you, and the CI passed.
Would you please help to check if the PR is OK to you?

**Review by XiaoningLiu — APPROVED** [review]

**Pull request merged**